cirq/ops/raw_types.py

```diff
@@ -510,9 +510,11 @@ def validate_args(self, qubits: Sequence['cirq.Qid']):
 
 @value.value_equality
 class TaggedOperation(Operation):
-    """A specific operation instance that has been identified with a set
-    of Tags for special processing.  This can be initialized with
-    Using Operation.with_tags(tag) or by TaggedOperation(op, tag).
+    """A specific operation instance that has identified with a set of Tags.
```

Comment:
Somehow "been" has been lost in translation (between "has" and "identified"). Also, this isn't really identifying the operation, but annotating it. Suggestion:

```python
"""Operation annotated with a set of Tags."""
```

Reply:
Done.